presto-docs/src/main/sphinx/include/PrestoThriftService.thrift

```diff
@@ -0,0 +1,305 @@
+namespace java.swift com.facebook.presto.connector.thrift.api
+
```

Comment:
Regarding formatting, here and in some of the cases below, this is what swift generator creates.